src/CommunityVoices/App/Website/Bootstrap/FrontController.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+namespace CommunityVoices\App\Website\Bootstrap;
+
+use CommunityVoices\Api\Component\Exception\AccessDenied;
+use CommunityVoices\Api\Component\Exception\MethodNotFound;
+
+/**
+ * @overview Front controller to bootstrap the CommunityVoices application together
+ * 
+ * Adapted from Fowler's POEAA: Front Controller
+ */
+
+class FrontController {
+    protected $router;
+    protected $dispatcher;
+    protected $injector;
+
+    /**
+     * Providers to initialize prior to request route & dispatching
+     * 
+     * Note that the routes provider is excluded because it sould be initialized
```

Comment:
*should

Reply:
thanks. I need tor remove that anyways since routes isn't a provider anymore